doc/reference/std/cli/getopt.md

```diff
@@ -0,0 +1,269 @@
+# Command Line Argument Parsing
+
+The `:std/cli/getopt` library provides facilities for command line argument parsing.
+
+::: tip usage
+```scheme
+(import :std/cli/getopt)
+```
+:::
+
+This library used to be available as `:std/getopt` up to Gerbil v0.18,
+and is still available under that name for now, but its use is deprecated.
```

Comment:
I think deprecate conveys the wrong intent here; the library itself is by no means deprecated, it just got moved.
Can we rephrase this?

Reply:
discouraged?